src/ast/mod.rs

```diff
@@ -426,11 +426,34 @@ impl fmt::Display for WindowFrameBound {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum ExplainLevel {
+    Analyze,
+    Verbose,
+}
+
+impl fmt::Display for ExplainLevel {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use self::ExplainLevel::*;
+        f.write_str(match self {
+            Analyze => "ANALYZE",
+            Verbose => "VERBOSE",
+        })
+    }
+}
+
 /// A top-level statement (SELECT, INSERT, CREATE, etc.)
 #[allow(clippy::large_enum_variant)]
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum Statement {
+    // EXPLAIN
+    Explain {
+        level: Option<ExplainLevel>,
```

Comment:
I think it's not an enum but rather a combination of options.
see: https://www.postgresql.org/docs/9.1/sql-explain.html

I think all the following are correct:

`EXPLAIN`
`EXPLAIN ANALYZE`
`EXPLAIN VERBOSE`
`EXPLAIN ANALYZE VERBOSE`

So maybe this would be a better encoding?
```rust
Explain {
    analyze: bool
    verbose: bool
}
```

Reply:
Yeap, you are right.

I checked What combintations work with PostgreSQL:

```
EXPLAIN VERBOSE SELECT * from test;
EXPLAIN ANALYSE SELECT * from test;
EXPLAIN ANALYZE VERBOSE SELECT * from test;
```

What doesnt work:

```
EXPLAIN VERBOSE ANALYZE SELECT * from test;
```

Find very funny thing, that they support `ANALYSE`, but it's not mentioned inside docs.